playbooks/deploy_dns.yml

```diff
@@ -2,6 +2,7 @@
   hosts: dns_host
   vars:
     SETUP_DNS_SERVICE: "{{ setup_dns_service | default(True) }}"
+    TFTP_ROOT: "{{ hostvars['tftp_host']['tftp_directory'] | default(/var/lib/tftpboot/) }}"
```

Comment:
I think it should be in the TFTP role and no in the DNS

Reply:
It is a tow different roles one is for the dnsmasq and the another one is for the TFTP role